src/core/Akka.API.Tests/verify/CoreAPISpec.ApproveCore.DotNet.verified.txt

```diff
@@ -4995,14 +4995,8 @@ namespace Akka.Util
     public sealed class FastLazy<T>
     {
         public FastLazy(System.Func<T> producer) { }
-        public bool IsValueCreated { get; }
-        public T Value { get; }
-    }
-    public sealed class FastLazy<S, T>
-    {
-        public FastLazy(System.Func<S, T> producer, S state) { }
-        public bool IsValueCreated { get; }
         public T Value { get; }
+        public bool IsValueCreated() { }
```

Comment:
Changed to method because with 
[MethodImpl(MethodImplOptions.AggressiveInlining)]
it shows slightly better performance